skopt/space.py

```diff
@@ -246,13 +264,23 @@ def __init__(self, categories, prior=None):
         self.categories = categories
         self.transformer = _CategoricalEncoder()
         self.transformer.fit(self.categories)
+        self._prior = prior
```

Comment:
Not sure actually. I thought we adopted the `sklearn` idea that each argument to the constructor becomes `self.name = value` and things which are computed from them `self.name_ = computed_value`. Right now it seems inconsistent (comparing `self.categories` and `self._low`). Let's see what @MechCoder's opinion is but I'd vote for the `sklearn` style even thought it might be a bit OTT.